src/libs/tools/timeline.c

```diff
@@ -917,9 +917,11 @@ static void _selection_collect(dt_lib_timeline_t *strip, dt_lib_timeline_mode_t
 
   // we construct the rule
   gchar *coll = NULL;
+  gboolean date_only = FALSE;
   if(strip->start_x == strip->stop_x)
   {
     coll = _time_format_for_db(strip->start_t, (strip->zoom + 1) / 2 * 2 + 2, FALSE);
+    date_only = strlen(coll) <= 10; //YYYY:MM:DD or shorter
```

Comment:
I would prefer having a static routine checking for date only. And maybe the best is to check for a space at the 11 position?

Reply:
The string could be just YYYY:MM, so we can't check coll[10] as that might be past the end of the string.
static function coming in 3...2...1...